packages/eas-cli/src/graphql/queries/credentials/AppleDeviceQuery.ts

```diff
@@ -0,0 +1,46 @@
+import assert from 'assert';
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../../client';
+import { AppleDevice, AppleDeviceFragment } from '../../types/credentials/AppleDevice';
+import { AppleTeam, AppleTeamFragment } from '../../types/credentials/AppleTeam';
+
+const AppleDeviceQuery = {
+  async getAllByAppleTeamIdentifierAsync(
+    accountId: string,
+    appleTeamIdentifier: string
```

Comment:
my memory is a bit fuzzy on this one. I noticed for the other queries and mutations, we use `appleTeamId`, but here we are using `appleTeamIdentifier`. Was there a reason we are using the identifier instead of the id?

Reply:
I originally created the `appleTeam.byAppleTeamIdentifier` query because I needed it for the device registration flow. When the user authenticates with Apple we only know their Apple Team Identifier. I needed to somehow resolve the Apple Team ID without fetching all the Apple Teams associated with an account (and filtering out the array on the client side).